test/Controllers/ServicesTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/*
+ * Services SDK Controller Test Case
+ */
+
+namespace GonebusyTest\Controllers;
+
+use PHPUnit\Framework\TestCase;
+
+use GonebusyLib\Configuration;
+use GonebusyLib\GonebusyClient;
+use GonebusyLib\Models\CreateServiceBody;
+use GonebusyLib\Models\UpdateServiceByIdBody;
+
+class ServicesTest extends TestCase
+{
+    /**
+     * To contain the GonebusyLib\GonebusyClient
+     */
+    protected $client;
+
+    /**
+     * To contain the GonebusyLib\Controllers\ServicesController
+     */
+    protected $services;
+
+    /**
+     * Create the GoneBusy SDK client for each test.
+     */
+    public function setUp() {
+        $this->client = new GonebusyClient();
+        $this->services = $this->client->getServices();
+    }
+
+
+    /**
+     * Returns arbitrary service data.
+     * @param  string $type Should be 'create' or 'update'.
+     * @return  CreateServiceBody or UpdateServiceByIdBody object with unique data to send to API
+     */
+    private function serviceBody($type) {
+        $rand = rand();
+        switch($type) {
+            case 'create':
+                return new CreateServiceBody(
+                    "description", // REQUIRED
+                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
+                    "name", // REQUIRED
+                    NULL, // categories
+                    NULL, // price_model_id XXX defaults to inexistent id
```

Comment:
Ok, I think you may have encountered a bug.  `price_model_id: 456` now appears but I had to assign an owner to it corresponding to this auth token.  The bug may be that default PriceModel created when creating Service is not getting associated with the owner correctly.  I'll log it and confirm with a spec.